lib/rpmsg/rpmsg_virtio.c

```diff
@@ -358,6 +358,19 @@ static void rpmsg_virtio_release_rx_buffer(struct rpmsg_device *rdev,
 	metal_mutex_release(&rdev->lock);
 }
 
+static int rpmsg_virtio_notify_wait(struct rpmsg_virtio_device *rvdev,
+				    struct virtqueue *vq)
```

Comment:
squash  declaration on 1 line (up to 100 char is OK)

Reply:
Ok, done